sessions/js-loops/stars/README.md

```diff
@@ -0,0 +1,73 @@
+# JS Loops: Stars
+
+Let's build an awesome interactive 5-star rating scale! The goal is to display filled / empty stars depending on the user input. A click on a star should fill all stars up to the clicked one and leave all others empty.
+
+## Task
+
+### Part 1: Render Stars
+
+Start by render 5 empty stars in the starContainer. For that, wirte the function 'renderStars':
+
+- Use a for loop inside the function to repeat the following code five times.
+  💡 Start the loop with the value 1, not with 0
+- Inside the for loop, create an image Element and set the "src" attribute to the path of the empty star image ("assets/star-empty.svg")
+- append the star to the "starContainer".
+
+### Part 2: Change the number of filled stars
+
+Now we want to render a certain number of filled stars, depending on the input of the function:
+
+- add an input parameter to the function called "filledStars"
```

Comment:
I was confused by the word "input parameter" because I rather thought about an input field / an event listener.

Should we rephrase here like
```suggestion
- refactor the function `renderStars` so that it now accepts a parameter called "filledStars"
```
or something similar?